embassy-stm32/src/rcc/f4/max.rs

```diff
@@ -0,0 +1,268 @@
+#[cfg(any(
+    feature = "stm32f401cb",
```

Comment:
it'd make sense to get `build.rs` to set a cfg like `stm32f401` by substringing the feature, so you can do `#[cfg(stm32f401)]`.